src/goto-symex/symex_main.cpp

```diff
@@ -46,7 +46,11 @@ symex_configt::symex_configt(const optionst &options)
     run_validation_checks(options.get_bool_option("validate-ssa-equation")),
     show_symex_steps(options.get_bool_option("show-goto-symex-steps")),
     max_field_sensitivity_array_size(
-      options.get_unsigned_int_option("max-field-sensitivity-array-size"))
+      options.is_set("no-array-field-sensitivity")
```

Comment:
We should produce an error if both are set

Reply:
I've added a check for that in jbmc/cbmc_parse_options